vllm/transformers_utils/config.py

```diff
@@ -498,8 +503,9 @@ def get_sentence_transformer_tokenizer_config(model: str,
             repo_files = list_repo_files(model,
                                          revision=revision,
                                          token=HF_TOKEN)
+            logger.info(repo_files)
```

Comment:
```suggestion
            logger.debug(repo_files)
```
I think it would be better to set debug level here.

Reply:
Thanks for the feedback.
I'll solve these pre-commit issues later. 😊